src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import './CommentList.scss';
+import { Comment } from '../../react-app-env';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList">
+    {!comments
+      ? ''
+      : comments.map((comment) => (
+        <CommentInfo key={comment.id} comment={comment} />
+      ))}
```

Comment:
maybe u can write something looks like this
```suggestion
    {comments && comments.map((comment) => (
      <CommentInfo key={comment.id} comment={comment} />
    ))}
```

Reply:
yes I can)